torch/utils/cpp_extension.py

```diff
@@ -1433,8 +1434,9 @@ def _prepare_ldflags(extra_ldflags, with_cuda, verbose, is_standalone):
             extra_ldflags.append('c10_cuda.lib')
         extra_ldflags.append('torch_cpu.lib')
         if with_cuda:
-            extra_ldflags.append('torch_cuda.lib')
-            # /INCLUDE is used to ensure torch_cuda is linked against in a project that relies on it.
+            extra_ldflags.append('torch_cuda_cu.lib')
+            extra_ldflags.append('torch_cuda_cpp.lib')
```

Comment:
should both these flags get their own -INCLUDE flag? I'm not too sure of what's going on here

Reply:
I have realized what is going on.